WordPress/src/main/java/org/wordpress/android/WordPressDB.java

```diff
@@ -430,13 +430,25 @@ public boolean addBlog(Blog blog) {
     }
 
     public List<Map<String, Object>> getBlogsBy(String byString, String[] extraFields) {
-        return getBlogsBy(byString, extraFields, 0);
+        return getBlogsBy(byString, extraFields, 0, true);
```

Comment:
Yeah, no other areas of the app would want to access the Jetpack blogs that were added using `FetchBlogListWPCom`. The auth token won't work for those sites so they should remain hidden everywhere except for Notification Settings.